fxmanifest.lua

```diff
@@ -1,18 +1,19 @@
 fx_version 'cerulean'
 game 'gta5'
 
-version '1.0.0'
+version '1.0.1'
```

Comment:
```suggestion
version '1.0.0'
```

We're not versioning yet.